scene/main/node.h

```diff
@@ -46,6 +46,8 @@ class Node : public Object {
 	GDCLASS(Node, Object);
 	OBJ_CATEGORY("Nodes");
 
+	friend class ResourceImporterScene;
```

Comment:
I need it to access `NameCasing`. Please, let me know if this is okay.

Reply:
Ok.